conf/distro/bisdn-linux.conf

```diff
@@ -32,6 +32,11 @@ PREFERRED_PROVIDER_virtual/docker = "docker-moby"
 # we do not have seccomp in DISTRO_FEATURES, so do not enable it for docker
 PACKAGECONFIG:pn-docker-moby ?= "docker-init"
 
+# disable sensord to avoid pulling in rrdtool and various graphical dependencies
+PACKAGECONFIG:pn-lmsensors ?= ""
+PACKAGECONFIG:pn-lmsensors-config ?= ""
+
 # use static uid/gid allocation to avoid renumeration on upgrade
 USERADDEXTENSION = "useradd-staticids"
 USERADD_ERROR_DYNAMIC = "error"
+
```

Comment:
Is this line added intentionally?

Reply:
Ah, no, must have been left when moved the options from the bottom when testing. Removed.